sources/packages/backend/libs/services/src/notifications/notification/gc-notify.model.ts

```diff
@@ -41,3 +41,21 @@ export interface GCNotifyErrorResponse {
   errors: GCNotifyErrorDetail[];
   status_code: HttpStatus;
 }
+
+/**
+ * Email message format.
+ */
+export interface EmailMessage {
```

Comment:
Event this being a `EmailMessage` inside GC Notify just wondering if it would be good to be explicit about the fact that this is notification specific `EmailMessage`, for instance, naming it as `NotificationEmailMessage`.